### ISSUE TITLE
Laggy resize of floating windows

### ISSUE THREAD

**Issue by substanceof** [baskerville/bspwm#716]
Video: https://streamable.com/hxark
bspwmrc: https://gist.github.com/substanceof/2359b4cafec97c2d5c9d7d147375820a
Move works well but resize (both side and corner) acts very weird.
This bug doesn't affect resize of tiled windows.

**Comment by msteen** [comment]
Did you try other windows than terminals? I also experience weird behavior when I try to resize a floating `xfce4-terminal` window, but with other windows I do not experience this problem. In the case of `xfce4-terminal` it seems to has to do with the recalculation of how many characters fit in the new width of the window, because I have the current working directory being shown at the right side, which it tries to render at the correct position after resizing.

**Comment by substanceof** [comment]
@msteen You're right. This bug affects terminal apps only. But on versions where mouse events was handled by sxhkd all seems to be normal. So its may be a regression.
